src/sentry/static/sentry/less/stream.less

```diff
@@ -516,32 +478,6 @@
 
   .issue {
     h3 {
-      .icon-star-solid,
-      .icon-soundoff {
-        padding-right: 5px;
-        font-size: 12px;
-        position: relative;
-        .transition(all 0.2s ease-in-out);
-        width: auto;
-      }
-
-      .icon-star-solid {
-        font-size: 14px;
-        top: -1px;
-        display: none;
-        color: @yellow-orange;
-        overflow: hidden;
-      }
-
-      .icon-soundoff {
```

Comment:
Wow you removed so many less css .. Kudos!! 👏👏

Reply:
I was surprised how many override rules these icons had.